akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ActiveActiveEventPublishingSpec.scala

```diff
@@ -129,14 +133,16 @@ class ActiveActiveEventPublishingSpec
         PersistenceId.replicatedUniqueId("myId4", "DC-B"),
         1L,
         "two",
-        System.currentTimeMillis())
+        System.currentTimeMillis(),
+        Some(VersionVector.empty))
       // simulate another published event from that replica
       actor.asInstanceOf[ActorRef[Any]] ! internal.PublishedEventImpl(
         Some("DC-B"),
         PersistenceId.replicatedUniqueId("myId4", "DC-B"),
         1L,
         "two-again", // ofc this would be the same in the real world, different just so we can detect
-        System.currentTimeMillis())
+        System.currentTimeMillis(),
+        Some(VersionVector.empty))
```

Comment:
I think we should couple the active active fields into a single Option field since they are always both or neither there (and make a class for them).

Reply:
👍